usage/app/lib/MediaUsageBuilder.scala

```diff
@@ -69,8 +72,27 @@ object MediaUsageBuilder {
       digitalUsageMetadata = None,
       syndicationUsageMetadata = None,
       frontUsageMetadata = Some(frontUsageRequest.metadata),
+      None,
       lastModified = frontUsageRequest.dateAdded
     )
   }
 
+  def build(downloadUsageRequest: DownloadUsageRequest, groupId: String): MediaUsage = {
+    val usageId = UsageIdBuilder.build(downloadUsageRequest)
+
+    MediaUsage (
+      usageId,
+      groupId,
+      downloadUsageRequest.mediaId,
+      DownloadUsage,
+      mediaType = "image",
+      downloadUsageRequest.status,
+      printUsageMetadata = None,
```

Comment:
Just out of interest – would it be possible/desirable to represent these as a sum type in some future refactor, rather than having `n` metadata fields for each usage type? It would reduce the number of lines changes when we added new metadata types.

Reply:
Yes, that would be ace! There would be two approaches to this, both with +/-:

- Change case class and database model and reindex database to ensure pre-existing data gets transformed into new shape
- Change case class, keep database model as is and incur the cost of the slight hidden complexity of different representations of data across domains

Either make sense, I think and worth the cost of a sum type as it yields simpler code.